widgets/src/layout/ignore_layout.rs

```diff
@@ -0,0 +1,98 @@
+use ribir_core::prelude::*;
+
+/// When the IgnoreLayoutChanged wrap the widget, the subtree's inner change
+/// will not notify to the parent, it mean's that parent will not do the
+/// re-layout. it's
```

Comment:
This comment `it's` next what? Or unnecessary?

Reply:
unnecessary